### ISSUE TITLE
Chore: Refactor docker-compose.yml

### ISSUE THREAD

**Issue by ntcho** [osamhack2021/Web_Handover_Handover#22]
### `docker-compose.yml` for Docker

- [x] Update to Compose file version 3.9
- [x] Compress `build` configuration with context

> `build` can be specified either as a string containing a path to the build context: (source: [here](https://docs.docker.com/compose/compose-file/compose-file-v3/#build))

```yml
version: "3.9"
services:
  webapp:
    build:
      context: ./dir
```

Above can be shortened like below:

```yml
version: "3.9"
services:
  webapp:
    build: ./dir
```

- [x] Decide whether to use `container_name` (they disable _scalability_)

> Because Docker container names must be unique, you cannot scale a service beyond 1 container if you have specified a custom name. Attempting to do so results in an error. (source: [here](https://docs.docker.com/compose/compose-file/compose-file-v3/#container_name))
- [x] Update `networks` configuration
- [x] Rewrite `volumes` configuration to [**long syntax**](https://docs.docker.com/compose/compose-file/compose-file-v3/#long-syntax-3) (along with `.gitignore`)

### `/.devcontainer` for GitHub Codespaces

- [x] Configure `docker-compose.yml` with `/.devcontainer` contents

**Comment by ntcho** [comment]
Referencing https://www.bezkoder.com/docker-mern/ this tutorial

**Comment by ntcho** [comment]
Resolved with #29.
